swap_meet/item.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+today = datetime.today()
+
+class Item():
+    def __init__(self, category = None, condition = 0, year = 0):
+        if category == None:
+            category = ""
+        self.category = category
```

Comment:
Since strings are immutable you don't need to use the `None` default argument trick:

```suggestion
    def __init__(self, category = "", condition = 0, year = 0):
        self.category = category
```